pkgs/tools/security/oath-toolkit/update.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i dash -p curl dash git nix
```

Comment:
```suggestion
#!nix-shell -i bash -p curl git gnugrep nix
```

bash is the default shell used by nix. No reason to use dash.